fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -225,7 +257,18 @@ def modifySceneTable(scene, exportInfo: ExportInfo):
         else:
             raise PluginError("ERROR: Scene not found in ``scene_table.h``!")
 
+    # get the new file data
+    newFileData = sceneTableToC(fileData, header, sceneNames, scene)
+
+    # apply HackerOoT changes if needed
+    isHackerOoT = False
+
+    with open(os.path.join(exportPath, "include/tables/scene_table.h"), "r") as file:
+        if '#include "config.h"' in file.read():
+            isHackerOoT = True
```

Comment:
that's not enough for checking HackerOoT, another project could very well have a `#include "config.h"` for whatever reason

Why not just use the "use hackeroot features" property?

Reply:
https://github.com/Fast-64/fast64/pull/203#issuecomment-1382182404

though I'll use that I guess :peepoShrug: if it's just me then I'm fine with using it

Reply:
well if the checkbox is there... you should expect users to have it checked

overall fast64 should not contain things that change behavior to accomodate forks of decomp, that enable on their own without user input
otherwise it'd get messy really fast